config.json.example

```diff
@@ -17,5 +17,6 @@
     "dbSchema": "./databases/_sponsorTimes.db.sql",
     "privateDBSchema": "./databases/_private.db.sql",
     "mode": "development",
-    "readOnly": false
+    "readOnly": false,
+    "webhooks": []
```

Comment:
add a webhook with a mock endpoint defined in the test code so the new feature is tested